lib/hooks/create/unique.js

```diff
@@ -34,6 +35,12 @@ module.exports = settings => {
       .where('status', '!=', 'new')
       .then(results => {
         if (results && results.length) {
+          if (type === 'project' && action === 'grant') {
```

Comment:
Might want to check if the open task is in an editable state here.

Reply:
good shout